xmtp/src/bin/update-schema.rs

```diff
@@ -37,7 +37,7 @@ fn update_schemas_encrypted_message_store() -> Result<(), std::io::Error> {
 
     {
         // Initalize DB to read the latest table definitions
-        let _ = EncryptedMessageStore::new(StorageOption::Peristent(tmp_db.clone()), EncryptedMessageStore::generate_enc_key() ).unwrap();
+        let _ = EncryptedMessageStore::new(StorageOption::Persistent(tmp_db.clone()), EncryptedMessageStore::generate_enc_key() ).unwrap();
```

Comment:
Unrelated to cross platform builds. Just fixes a typo I came across while digging through things. Can be split into separate PR if desired.